.github/workflows/repo-automator.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Repo Automator'
+on:
+  issues:
+    types:
+      - opened
+  push:
+    branches:
+      - develop
```

Comment:
Curious if there's any benefit to running this when changes are merged into `develop`? Seems like this action is for Issues and PRs so doesn't seem like this would be needed.

Reply:
@dkotter Yes, Action is for handling issues and PRs, but it also identifies potential merge conflicts with PR branches. So, essentially, it is related to PRs, but we need to trigger this on the push to the `develop` branch to check potential merge conflicts and auto-label/auto-comment PRs with merge conflicts.

Action internally [handle operations](https://github.com/10up/action-repo-automator/blob/develop/index.js#L15) based on the trigger. So, it will only run the merge conflicts check when trigger is push to develop and PR/Issue related check will be skipped.

Please let me know if you need any further information on this.